packages/babel-traverse/src/path/index.js

```diff
@@ -23,29 +23,29 @@ import * as NodePath_comments from "./comments";
 
 const debug = buildDebug("babel");
 
+export const REMOVED = 1 << 0;
+export const SHOULD_STOP = 1 << 1;
+export const SHOULD_SKIP = 1 << 2;
```

Comment:
Some great work doing this research!

I would just like to note that using bitwise flags and operators is for sure going to make contributing and reading the coding that much more difficult (for me, and I will assume some others). I think I mentioned it earlier on Slack and it's a hard call but trading off readability/performance. I don't know what else we can do other than writing comments to explain things though